plugin/src/openshift/utils/KialiIntegration.tsx

```diff
@@ -124,6 +156,27 @@ export const useInitKialiListeners = (): void => {
             consoleUrl = `/k8s/ns/${namespace}${istioUrl}/${OSSM_CONSOLE}${webParams}`;
           }
         }
+      } else if (kialiAction.startsWith('/tracing')) {
+
+        if (distributedTracingPluginConfig && distributedTracingPluginConfig.extensions.length > 0 && pluginConfig) {
+            const urlParams = new URLSearchParams(kialiAction.split('?')[1]);
+            const namespace = pluginConfig.observability.namespace;
+            const instance = pluginConfig.observability.instance;
+            const trace = urlParams.get('trace');
+            const tenant = pluginConfig.observability.tenant ? pluginConfig.observability.tenant : "default"
```

Comment:
Added Semicolon and simpler structure
```suggestion
            const tenant = pluginConfig.observability.tenant ?? "default";
            
```